samples/drumthumper/src/main/java/com/google/oboe/sample/drumthumper/DrumThumperActivity.kt

```diff
@@ -83,10 +81,15 @@ class DrumThumperActivity : AppCompatActivity(), TriggerPad.DrumPadTriggerListen
             var pad: TriggerPad = findViewById(R.id.crashPad)
             pad.addListener(this)
         }
+
+        mDrumPlayer.setupAudioStream()
+        mDrumPlayer.loadWavAssets(getAssets())
     }
 
     override fun onPause() {
         super.onPause()
+
+        mDrumPlayer.teardownAudioStream()
```

Comment:
onStart() and onPause are not balanced.
You might get onPause() followed by onResume() and then the audio will be dead.